doc/eID/eid-integration-magnus.md

```diff
@@ -0,0 +1,37 @@
+# UseID Integration
+
+To integrate UseID and mobile app and web widget, the following steps are necessary for services already having an existing eID integration.
+
+## Integration
+
+### 1. Embedding Widget
+
+eService embeds widget via JavaScript by adding the following empty container to the HTML page:
+
+```html
+<div id="useid-widget-container" data-tc-token-url="{}"></div>
+```
+
+and including this script in the page:
+
+```html
+<script async src="https://useid.ds4g.net/widget.js"></script>
+```
+
+As value for `data-tc-token-url` the already known tcTokenURL is used.
+
+> Previously, here was a link including the tcTokenURL as query parameter for the user to click to open the app. Now a widget is displayed, which can be scanned by the user with the smartphone to start the eID scanning in the UseID mobile app.
```

Comment:
I would rephrase it to something like: 
"The eService usually presents a link to the user which points to the eID client application (e.g. AusweisApp2) including the tcTokenURL as a query parameter. With UseID, this is replaced by the web widget displaying a QR code which can be scanned to start the UseID mobile app."

Reply:
Sound way better! 👍